content/c-sharp/concepts/math-functions/terms/pow/pow.md

```diff
@@ -0,0 +1,46 @@
+---
+Title: '.Pow()'
+Description: 'Calculates the first input number to the power of the second input number.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Methods'
+  - 'Numbers'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`Math.Pow()`** method returns the first input number raised to the power of the second input number.
+
+## Syntax
+
+```pseudo
+Math.Pow(num1, num2)
+```
+
+`Math.Pow()` returns the calculated value of `num1` to the power of `num2`.
```

Comment:
```suggestion
`Math.Pow()` takes two values of type `double`. It will return a `double` that is the result of the value `num1` raised to the power of value `num2`.
```